iogt/static/css/_base.scss

```diff
@@ -29,3 +34,16 @@ h1, h2, h3, h4, h5, h6 {
   font-size: 16px;
```

Comment:
Why do we have font settings in an `img` tag? Does this affect the `alt` property?